<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>1068</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Data Capture</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
            <property name="spacing">
             <number>6</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,1">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>233</height>
                 </size>
                </property>
                <property name="title">
                 <string>Acquisition Control</string>
                </property>
                <widget class="QPushButton" name="AcqDataButton">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>20</y>
                   <width>91</width>
                   <height>23</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Acquire Data</string>
                 </property>
                </widget>
                <widget class="QCheckBox" name="StopChBox">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>50</y>
                   <width>91</width>
                   <height>21</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Stop</string>
                 </property>
                </widget>
                <widget class="QCheckBox" name="SingleRetriggerChBox">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>70</y>
                   <width>91</width>
                   <height>21</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Single</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="acquisitionnumberle">
                 <property name="geometry">
                  <rect>
                   <x>80</x>
                   <y>120</y>
                   <width>113</width>
                   <height>20</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_2">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>120</y>
                   <width>71</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Acquisitions</string>
                 </property>
                </widget>
                <widget class="QCheckBox" name="MultiAcquisitionChBox">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>90</y>
                   <width>101</width>
                   <height>21</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Multi Acquisition</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_3">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>150</y>
                   <width>71</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Points</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="userdatapointsle">
                 <property name="geometry">
                  <rect>
                   <x>80</x>
                   <y>150</y>
                   <width>113</width>
                   <height>20</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>100000</string>
                 </property>
                </widget>
                <widget class="QCheckBox" name="ScreenshotChBox">
                 <property name="geometry">
                  <rect>
                   <x>140</x>
                   <y>50</y>
                   <width>111</width>
                   <height>21</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>CollectScreenshot</string>
                 </property>
                </widget>
                <widget class="QProgressBar" name="datatransferprogress">
                 <property name="geometry">
                  <rect>
                   <x>30</x>
                   <y>190</y>
                   <width>231</width>
                   <height>23</height>
                  </rect>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="SaveOptionsGrpBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>134</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="title">
                 <string>Save Options</string>
                </property>
                <widget class="QCheckBox" name="addtimestampchbox">
                 <property name="geometry">
                  <rect>
                   <x>150</x>
                   <y>100</y>
                   <width>101</width>
                   <height>21</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Add Timestamp</string>
                 </property>
                </widget>
                <widget class="QLabel" name="filenamelabel">
                 <property name="geometry">
                  <rect>
                   <x>20</x>
                   <y>70</y>
                   <width>71</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>File Name</string>
                 </property>
                </widget>
                <widget class="QLabel" name="label_4">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>20</y>
                   <width>71</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Save Path</string>
                 </property>
                </widget>
                <widget class="QCheckBox" name="allowoverwritechbox">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>100</y>
                   <width>101</width>
                   <height>21</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Allow Overwrite</string>
                 </property>
                </widget>
                <widget class="QLineEdit" name="filenamele">
                 <property name="geometry">
                  <rect>
                   <x>100</x>
                   <y>70</y>
                   <width>161</width>
                   <height>20</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>100000</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="selectsavepathbut">
                 <property name="geometry">
                  <rect>
                   <x>80</x>
                   <y>20</y>
                   <width>91</width>
                   <height>23</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Select</string>
                 </property>
                </widget>
                <widget class="QLabel" name="savepathlabel">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>50</y>
                   <width>281</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Save Path</string>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>114</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="title">
                 <string>Connection Info and Log</string>
                </property>
                <widget class="QPushButton" name="ConnectButton">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>80</y>
                   <width>75</width>
                   <height>20</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Connect</string>
                 </property>
                </widget>
                <widget class="QComboBox" name="scopelistcombox">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>50</y>
                   <width>271</width>
                   <height>22</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QLabel" name="label">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>20</y>
                   <width>61</width>
                   <height>16</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Scope List</string>
                 </property>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="QTextBrowser" name="InfoText">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>300</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PlotWidget" name="MainPW" native="true">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Data Viewer</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="title">
                 <string>GroupBox</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="title">
                 <string>GroupBox</string>
                </property>
                <widget class="QPushButton" name="AcqDataButton_2">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>210</y>
                   <width>75</width>
                   <height>23</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Acquire Data</string>
                 </property>
                </widget>
                <widget class="QTextBrowser" name="InfoText_2">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>260</y>
                   <width>281</width>
                   <height>341</height>
                  </rect>
                 </property>
                </widget>
                <widget class="QPushButton" name="ConnectButton_2">
                 <property name="geometry">
                  <rect>
                   <x>10</x>
                   <y>180</y>
                   <width>75</width>
                   <height>23</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>Connect</string>
                 </property>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="PlotWidget" name="MainPW_2" native="true">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
